.env

```diff
@@ -0,0 +1,5 @@
+REACT_APP_BASE_URL=""
+
+REACT_APP_AUTH0_DOMAIN = "dev-xsron5cxoe45bn57.us.auth0.com"
+REACT_APP_AUTH0_CLIENT_ID = "zmgQ6uO27tIDiDuJjGC4ZF0M3l0OySlv"
+REACT_APP_AUTH0_CLIENT_SECRET = "zLPnkkJHxuNwru0tZDVUDeBjcQxnJJEStnS4o3GZk3z4D-m73_vYikMR6Sk3X0kL"
```

Comment:
The .env file cannot be sent to git because sensitive data is stored there. Put this file into the gitignore config